examples/bridge-app/linux/main.cpp

```diff
@@ -711,6 +738,10 @@ Protocols::InteractionModel::Status emberAfExternalAttributeWriteCallback(Endpoi
         {
             ret = HandleWriteOnOffAttribute(static_cast<DeviceOnOff *>(dev), attributeMetadata->attributeId, buffer);
         }
+        else if ((dev->IsReachable()) && (clusterId == BridgedDeviceBasicInformation::Id))
+        {
+            ret = HandleWriteBridgedDeviceBasicAttribute(static_cast<Device *>(dev), attributeMetadata->attributeId, buffer);
```

Comment:
no need for casting, since `dev` is already a Dev * 

```suggestion
            ret = HandleWriteBridgedDeviceBasicAttribute(dev, attributeMetadata->attributeId, buffer);
```

Reply:
Done